.github/workflows/pythonpublish.yml

```diff
@@ -1,8 +1,8 @@
 # To create a release:
-#  - update the version number in g2p/_version.py
 #  - commit the changes
 #  - create an annotated tag with the version number, e.g.: git tag -a v2.0.1 -m "v2.0.1"
-#  - push the tag, which will trigger this pythonpublish release workflow
+#  - push the tag, which will trigger this pythonpublish release workflow and hatch build will build and publish using the tag version
+#     - Make sure that the tagged commit is the last commit, otherwise a dev version will be published
```

Comment:
could you have a situation where you tag a commit, then make another commit, then push everything including tags? I haven't actually tested that out, but if I build locally in that state I will get a dev version

Reply:
Yes, but you can't get to that state in the action workflow itself, because of 
```
on:
  push:
    tags:
      - v[0-9]+.**
```

Reply:
I guess if you're worried we might manually publish, then this warning would be meaningful in that case. Maybe qualify the warning, since "if you're trying to publish manually, ..." but I'd rather have a warning that says x-nay on publishing manually, no?